Switch to back-ticks for code block

alfarolabs opened the pull request:
…ish.md

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

alfarolabs committed:
Update adjust-the-height-of-an-element-using-the-height-property.spanish.md

RandellDawson reviewed (changes requested):
When using the 3 backticks, the line before and after the backticks needs to be blank for them to render correctly.  Always make sure to look at the preview before submitting your PR.  Also, can you please update the text of the tests for the parts below:
```
Your code should change the <code>h4</code> <code>height</code> property to a value of 25 pixels.
```

Thank you

alfarolabs committed:
Updated test text to Spanish

And changed CSS rule back to English in the example. Using back-ticks to properly format the code block
